api/config.json.example

```diff
@@ -8,6 +8,7 @@
     "smtp" : {
       "host": "mail.example.com",
       "port": 587,
+      "starttls": true,
```

Comment:
Could you default to 465 and Standard TLS, as it is preferable over STARTTLS?

Reply:
done as requested